Makefile

```diff
@@ -0,0 +1,24 @@
+# File: Makefile
+SHELL           = /bin/bash
+PROJECT         = thesis
+DIRECTORY			  = -output-directory build
+
+.PHONY          = all clean
+
+.PRECIOUS: $(PROJECT).ps
+
+all: pdf
+
+pdf: $(PROJECT).pdf
+
+view: $(PROJECT).pdf
+	acroread $(<)
```

Comment:
reference to acroread?
Personally, I would just remove the view target, actually.

Reply:
Oh I completely missed that. Yes that should have been deleted.